vnext/RNTesterCopy/js/FocusEventsExample.js

```diff
@@ -1,217 +0,0 @@
-/**
```

Comment:
We don't reference this file in our RNTesterList or anywhere in our project, so guessing it was Mac only.